src/contributing.md

```diff
@@ -116,32 +90,77 @@ make a documentation change, add
 to the end of the pull request description, and [@rust-highfive][rust-highfive] will assign
 [@steveklabnik][steveklabnik] instead of a random person. This is entirely optional.
 
+In addition to being reviewed by a human, pull requests are automatically tested
+thanks to continuous integration (CI). Basically, every time you open and update
+a pull request, CI builds the compiler and tests it against the
```

Comment:
```suggestion
a PR, CI builds the compiler and tests it against the
```

Reply:
This avoids having to decide between *pull request* and *Pull Request*

Reply:
I don't think Pull Request vs pull request is a problem however

Reply:
The inconsistency bothered me